packages/coachmark/README.md

```diff
@@ -25,9 +24,11 @@ When looking to leverage the `Coachmark` base class as a type and/or for extensi
 import { Coachmark } from '@spectrum-web-components/coachmark';
 ```
 
-## Example
+### Examples
```

Comment:
Can we make this an "Anatomy" and "Options" sections to make it consistent with the structure of other revised docs. Something that outlines the parts of the coachmark and how to add them.

See https://ttomar-coachmark-docs-audit--spectrum-wc.netlify.app/components/dialog/#anatomy and https://ttomar-coachmark-docs-audit--spectrum-wc.netlify.app/components/alert-banner/#anatomy